src/protocols/http/hpr_http_roaming_downlink_handler.erl

```diff
@@ -92,5 +92,5 @@ handle_event(Event, _Data, _Args) ->
 ) -> ok.
 send_response(ResponseStream, DownlinkPacket) ->
     lager:debug("sending response: ~p, pid: ~p", [DownlinkPacket, ResponseStream]),
```

Comment:
This wasn't changed, but I don't think we should be logging the whole downlink packet here. 
Log aggregator getting upset about binaries and whatnot.

It might be nice to have a log message about a downlink being sent from which protocol.

Reply:
Done.